src/Tizen.NUI/src/internal/Common/ProcessorController.cs

```diff
@@ -36,41 +33,39 @@ namespace Tizen.NUI
     /// </summary>
     internal sealed class ProcessorController : Disposable
     {
-        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        internal delegate void ProcessorCallback();
-
-        private ProcessorCallback callback = null;
-
-        public event EventHandler ProcessorOnceEvent;
-
-        public event EventHandler ProcessorEvent;
-
-        public event EventHandler LayoutProcessorEvent;
-
-        private ProcessorController() : this(Interop.ProcessorController.New(), true)
-        {
-        }
-
-        internal ProcessorController(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
-        {
-            callback = new ProcessorCallback(Process);
-            Interop.ProcessorController.SetCallback(SwigCPtr, callback);
-        }
-
         private static ProcessorController instance = null;
 
         public static ProcessorController Instance
         {
             get
             {
-                if(instance == null)
+                if (instance == null)
                 {
                     instance = new ProcessorController();
                 }
                 return instance;
             }
         }
 
+        private ProcessorController() : this(Interop.ProcessorController.New(), true)
```

Comment:
It seems that Properties are written after Constructors based on NUI coding rule.

Reply:
![image2020-10-28_14-24-37](https://user-images.githubusercontent.com/29364140/131493326-f2ebf2de-5381-4880-85e6-3c4caa9f2cfc.png)

Reply:
done